src/test/specific_tests.py

```diff
@@ -58,6 +58,13 @@ def test_zfin_gene_has_expression_link():
     for record in result:
         assert record["counter"] > 0
 
+def test_mods_have_gene_expression_atlas_link():
+    query = "MATCH (g:Gene)-[]-(c:CrossReference) " \
+            "WHERE c.crossRefType = 'gene/expression-atlas' " \
+            "RETURN count(distinct(g.taxonId)) AS counter"
+    result = execute_transaction(query)
+    for record in result:
+        assert record["counter"] > 0
```

Comment:
could this assert the number of taxonIds you expect to have links?  That way if one MOD was missing, we'd know right away?

Reply:
I am currently running all these suggestions. I had it = to 7 for this test before but wasn't sure if it would break tests in the test dataset. I am pretty sure it is fine to have the exact number. So I am changing back as I initially intended the test. Thanks

Reply:
Sounds great! - can you also check how long the entire build takes locally now too?  I think its just right around an 1.17 hours for me locally, pre atlas additions?

Reply:
Ya, it was around that long. I didn't measure it exactly but I had run it a couple of times today all the way thought. Once on master and then once on this branch. I was running it while we were in the architecture meeting. So zoom was definitely competing for resources. But your 1.17 hours seams approximately right.